clang/lib/3C/AVarBoundsInfo.cpp

```diff
@@ -873,6 +873,15 @@ AVarBoundsInfo::handlePointerAssignment(clang::Stmt *St, clang::Expr *L,
   return true;
 }
 
+void
+AVarBoundsInfo::mergeBoundsKey(BoundsKey To, BoundsKey From) {
+  if (InvalidBounds.find(To) != InvalidBounds.end() ||
+      InvalidBounds.find(From) != InvalidBounds.end()) {
+    InvalidBounds.insert(To);
+    InvalidBounds.insert(From);
+  }
+}
```

Comment:
So: Merging bounds keys means making the bounds of both invalid. Fine. But seems a little limiting? E.g., if you have bounds `count(10)` and bounds `count(5)` you could make it `count(5)` right?